lib/docs/user/rag.md

```diff
@@ -1 +1,64 @@
-# TODO
+# Automated Document Vector Store Ingestion Pipeline
+
+## Overview
+The Automated Document Ingestion Pipeline is designed to enhance LISA's RAG capabilities. This feature provides customers with a flexible, scalable solution for loading documents into configured vector stores. Customers have two methods to load files into vector stores configure with LISA. Customer can either manually load files via the chatbot user interface (UI), or via an ingestion pipeline. Files loaded via the chatbot UI are limited by Lambda's service limits on document file size and volume. Documents loaded via a pipeline are not subject to these limits, further expanding LISA’s ingestion capabilities. This pipeline feature supports the following document file types: PDF, docx, and plain text. The individual file size limit is 50 MB.
```

Comment:
- Customers have two methods to load files into vector stores ~~configure~~ **configured** with LISA.
- ~~Customer~~ **Customers** can either manually load files via the ch